pkg/dockerregistry/server/auth.go

```diff
@@ -157,6 +167,18 @@ func newAccessController(options map[string]interface{}) (registryauth.AccessCon
 
 	tokenRealm, _ := options[TokenRealmKey].(string)
 
+	if v, ok := options[ClientRetriesKey].(int); ok {
+		clientRetries = v
+	}
+
+	if v, ok := options[ClientDelayKey].(string); ok {
```

Comment:
You can use [`getDurationOption`](https://github.com/openshift/origin/blob/master/pkg/dockerregistry/server/util.go#L85).